specs/deneb/p2p-interface.md

```diff
@@ -264,26 +255,21 @@ to be fully compliant with `BlobsSidecarsByRange` requests.
 participating in the networking immediately, other peers MAY
 disconnect and/or temporarily ban such an un-synced or semi-synced client.
 
-Clients MUST respond with at least the first blobs sidecar that exists in the range, if they have it,
-and no more than `MAX_REQUEST_BLOCKS_DENEB` sidecars.
+Clients MUST respond with at least the first blobs sidecar that exists in the range, if they have it, and no more than `MAX_REQUEST_BLOB_SIDECARS * MAX_BLOBS_PER_BLOCK` sidecars.
```

Comment:
ah, we can get into a bad situation in the event that your peers only respond with one sidecar. Say there are 4 blobs per block, then the first query will give you index 0. Then if you query again, you would again be given index 0 for that same slide. There is no (current) way to advance through all blobs if you get responses of size < `MAX_BLOBS_PER_BLOCK`

The two methods to fix would be to be able to ask for a `start_index` in addition to `start_slot` OR to make it so that they MUST respond with at least the first `MAX_BLOBS _PER_BLOCK ` sidecars

Reply:
24a19bb886f7294f8b9c8ff990818936b044f5fa - a variation on the last point, where they must respond with all blobs of any block they start including (since the problem repeats for 3-blob blocks, the second block and so on) - this is what consumers want really when they're syncing ranges, ie full blocks, and anyone not giving that is just being difficult